src/molecules/Tabs.tsx

```diff
@@ -94,7 +93,7 @@ const Tabs: React.FC<TabsPros> = ({
           exact
           render={() => (
             <StyledP>
-              <Description dataDescription={description} />
+              <p>{description} </p>
```

Comment:
I recall you've used the `Description` component somewhere else too, no? Try to check if you have `Description` component rendered somewhere. But since the project builds I guess there is no problem.